core/rego.go

```diff
@@ -33,6 +33,7 @@ func newRegoInstanceBuilder(policy string, opaModuleConfig *OPAModuleConfig, eva
 	if evaluatorOptions == nil {
 		evaluatorOptions = &OPAEvaluatorOptions{}
 	}
+	evalOpts := evaluatorOptions
```

Comment:
Do you really need to introduce this new variable sharing the same reference with the other one?